changelog.txt

```diff
@@ -1,5 +1,9 @@
 *** Sensei Share Your Grade Changelog ***
 
+2020.04.02 - version 2.0.1
+* Tweak: Remove call to `user_started_course` preparing to Sensei LMS 3.0 - #41
```

Comment:
Can we put new first followed by tweaks? Also, it will read better if we change this to say `Tweak: Remove call to user_started_course`

Reply:
Fixed https://github.com/woocommerce/sensei-share-your-grade/pull/43/commits/a701658196951449d47b93cd0f9e1b83a641f5ff